Documentation for AIA (#26940)
keycloak/keycloak: docs/documentation/server_admin/topics/users/con-required-actions.adoc

You can set the actions that a user must perform at the first login. These actions are required after the user provides credentials. After the first login, these actions are no longer required. You add required actions on the *Details* tab of that user.

> Some required actions are automatically triggered for the user during login even if they are not explicitly added to this user by administrator. For example `Update password` action can be

Reviewer: ```suggestion
Some required actions are automatically triggered for the user during login even if they are not explicitly added to this user by the administrator. For example `Update password` action can be
```

> triggered in case that <<_password-policies, Password policies>> are configured in a way that user password need to be changed every X days. Or `verify profile`

Reviewer: ```suggestion
triggered if <<_password-policies, Password policies>> are configured in a way that the user password needs to be changed every X days. Or `verify profile`
```

> action can require user to update his <<user-profile, User profile>> as long as some user attributes do not match the requirements according to the user profile configuration.

Reviewer: ```suggestion
action can require the user to update the <<user-profile, User profile>> as long as some user attributes do not match the requirements according to the user profile configuration.
```
